saffron/benches/read_proof_bench.rs

```diff
@@ -0,0 +1,117 @@
+//! Run this bench using `cargo criterion -p saffron --bench read_proof_bench`
+
+use ark_ff::{One, UniformRand, Zero};
+use ark_poly::{univariate::DensePolynomial, Evaluations};
+use criterion::{black_box, criterion_group, criterion_main, BatchSize, Criterion};
+use kimchi::{circuits::domains::EvaluationDomains, groupmap::GroupMap};
+use mina_curves::pasta::{Fp, Vesta};
+use once_cell::sync::Lazy;
+use poly_commitment::{commitment::CommitmentCurve, ipa::SRS, SRS as _};
+use rand::rngs::OsRng;
+use saffron::{
+    env,
+    read_proof::{prove, verify},
+    ScalarField, SRS_SIZE,
+};
+
+// Set up static resources to avoid re-computation during benchmarks
+static SRS: Lazy<SRS<Vesta>> = Lazy::new(|| {
```

Comment:
Maybe using `pub fn get_srs_test<G>() -> SRS<G>` from `kimchi/src/precomputed_srs.rs` would be a more canonical way?

Reply:
Seems ok, but only because we have the same size. I was first thinking about making the change, but wondering now if it is the best as Saffron and Kimchi are too different protocols. We might want to differentiate them?